test/Conversion/allocate_shared_memory.mlir

```diff
@@ -1,6 +1,6 @@
 // RUN: triton-opt %s --allocate-shared-memory | FileCheck %s
 
-#blocked = #ttg.blocked<{sizePerThread = [32, 32], threadsPerWarp = [32, 1], warpsPerCTA = [4, 1], order = [1, 0]}>
+#blocked = #ttg.blocked<{sizePerThread = [1, 1], threadsPerWarp = [32, 1], warpsPerCTA = [2, 2], order = [1, 0]}>
```

Comment:
Is this change relevant?

Reply:
Yes, now that this layout can trigger the better codegen path, I have to alter it to make sure it triggers the smem path